app/src/lib/shells/win32.ts

```diff
@@ -52,6 +57,20 @@ export async function getAvailableShells(): Promise<
     })
   }
 
+  const hyper = await readRegistryKeySafe(
+    'HKEY_CURRENT_USER\\Software\\Classes\\Directory\\Background\\shell\\Hyper\\command'
+  )
+  if (hyper.length > 0) {
+    const commandPieces = hyper[0].value.match(/(["'])(.*?)\1/)
```

Comment:
Would you mind adding a comment in here about why we're looking at this registry path, and why we're extracting a specific value, for future reference?

Reply:
Added in a2241c168bc97efa4d25061c7e6e2310eac34b46. 😃